docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md

```diff
@@ -245,3 +245,10 @@ with `cert-manager.io/v1`
   ```yaml
   serviceAccountName: manager
   ```
+
+## Percentage String or Int API input will fail with a string different from an integer with % appended.
+
+`MachineDeployment.Spec.Strategy.RollingUpdate.MaxSurge`, `MachineDeployment.Spec.Strategy.RollingUpdate.MaxUnavailable` and `MachineHealthCheck.Spec.MaxUnhealthy` would have previously taken a String value with an integer character in it e.g "3" as a valid input and process it as a percentage value.
```

Comment:
this doc is meant for infra providers & developers, I think we also need to warn users. I suggest adding a warning note (using <aside>) to https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/src/tasks/upgrading-clusters.md#upgrading-machines-managed-by-a-machinedeployment (I couldn't find anywhere else with actual docs on rolling updates, I think that's missing from docs)

We should also make sure to include this info in the release notes when we release 0.4

Reply:
Added to the other doc as well.
Do we have an automated way to add release notes?

Reply:
Adding the ⚠️ in the PR title will automatically trigger a breaking change release note